sklearn/tree/_splitter.pyx

```diff
@@ -816,20 +945,46 @@ cdef class DensePartitioner:
         SIZE_t best_pos,
         double best_threshold,
         SIZE_t best_feature,
+        SIZE_t best_n_missing,
     ) noexcept nogil:
         """Partition samples for X at the best_threshold and best_feature."""
```

Comment:
Is it possible to indicated 
```suggestion
        """Partition samples for X at the best_threshold and best_feature.

        If missing values are present,  this method partitions `samples`
        so that the `best_n_missing` missing values' indices are the right
        most end of `samples` that is `samples[end_non_missing:end]`.
        """
```